### ISSUE TITLE
The stats page does not display statistics correctly

### ISSUE THREAD

**Issue by Martin22** [Freika/dawarich#547]
Hi,

On the statistics page, the country and city statistics are not displayed correctly. After clicking on a specific year, the same issue occurs with individual months, but when a specific month is selected, the countries and cities are displayed correctly.

The data is imported from the Google Semantics History export.

**Version** 0.19.7

![image](https://github.com/user-attachments/assets/394bd76f-a54e-4210-aafd-d218044a7c10)
![image](https://github.com/user-attachments/assets/d20999de-545c-4de5-93f1-b3e18b42ee01)
![image](https://github.com/user-attachments/assets/6f650a8b-5e07-400c-b15f-9f046800fbaa)

**Comment by nebriv** [comment]
Also having this issue.

**Comment by tabacha** [comment]
The stats need to be updated. This can be done by clicking on "Update stats" or by just waiting an hour. When the stats are read, you will get an notification. Do you get this notification? If not, please send your docker file. Perhaps the sidkiq container is killed cause of not enough RAM.

**Comment by nebriv** [comment]
Yeah I get the notification that the stats are updated however there is no change in the stats for me. Definitely looks similar to 406 for sure. I'll dig through the logs and see if anything stands out. 

![image](https://github.com/user-attachments/assets/63a884af-edd8-43fd-aad3-f349a663ae0c)

![image](https://github.com/user-attachments/assets/e087dac4-92ae-425f-9b36-581c9a7e0495)

**Comment by nebriv** [comment]
Clearing the stats as defined in this comment resolved the issue for me.
https://github.com/Freika/dawarich/issues/271#issuecomment-2369342062

**Comment by leeaash** [comment]
I got the same issue on 0.29.1, even after I did as https://github.com/Freika/dawarich/issues/271#issuecomment-2369342062 .

**Comment by Freika** [comment]
Should be fixed in 0.36.3
